src/SameAsInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3;
+
+interface SameAsInterface
```

Comment:
This time I would not make an interface, there is only going to be one implementation.